drivers/volume/volume.go

```diff
@@ -751,3 +751,81 @@ func IsCSIDriverWithoutSnapshotSupport(pv *v1.PersistentVolume) bool {
 	}
 	return true
 }
+
+// GetNodeRegion - get the node region
+func GetNodeRegion() (string, error) {
+	nodes, err := core.Instance().GetNodes()
+	if err != nil {
+		logrus.Errorf("getNodeZones: getting node details failed: %v", err)
+		return "", fmt.Errorf("failed in getting the nodes: %v", err)
+	}
+	for _, node := range nodes.Items {
+		return node.Labels[v1.LabelTopologyRegion], nil
+	}
+	return "", fmt.Errorf("failed in getting the nodes: %v", err)
+}
+
+// GetNodeZones - get zone details of the nodes in the cluster.
+func GetNodeZones() ([]string, error) {
+	nodes, err := core.Instance().GetNodes()
+	if err != nil {
+		logrus.Errorf("getNodeZones: getting node details failed: %v", err)
+		return nil, fmt.Errorf("failed in getting the nodes: %v", err)
+	}
+	var nodeZoneList []string
+	for _, node := range nodes.Items {
+		nodeZone := strings.Split(node.Labels[v1.LabelTopologyZone], "-")
+		nodeZoneList = append(nodeZoneList, nodeZone[2])
+	}
+	return nodeZoneList, nil
+}
+
+// GetVolumeBackupZones - Get zone details from the volumeBackupInfo
+func GetVolumeBackupZones(
+	volumeBackupInfos []*storkapi.ApplicationBackupVolumeInfo,
+) []string {
+	var backupZoneList []string
+	for _, volume := range volumeBackupInfos {
+		backupZone := strings.Split(volume.Zones[0], "-")
+		backupZoneList = append(backupZoneList, backupZone[2])
```

Comment:
same here: shouldn't this be a map instead  so we that have unique list of zones?